llvm/include/llvm/IR/IntrinsicsPowerPC.td

```diff
@@ -210,6 +210,15 @@ let TargetPrefix = "ppc" in {  // All intrinsics start with "llvm.ppc.".
             [llvm_float_ty],
             [llvm_float_ty, llvm_float_ty, llvm_float_ty, llvm_vararg_ty],
             [IntrNoMem]>;
+  // Load of a value provided by the system library at a fixed address. Used for
+  // accessing things like HWCAP word provided by GLIBC. The immediate argument
```

Comment:
Minor nits:
- Should it be `the HWCAP word`?
- We use GLIBC on this line but Glibc on line 217. I think it would be better if we were consistent and use one way of addressing GLIBC.

Reply:
I'm fine with both of these suggestions.